File: platforms/swift/examples/scene/camera/position.swift
import FragmentColor

let camera = try Camera.perspective(1.047, 16.0 / 9.0, 0.1, 100.0).lookAt([3.0, 2.0, 8.0], [0.0, 0.0, 0.0], [0.0, 1.0, 0.0])

let eye = camera.position()

File: platforms/kotlin/examples/scene/camera/position.kt
import org.fragmentcolor.*

val camera = Camera.perspective(1.047f, 16.0f / 9.0f, 0.1f, 100.0f).lookAt(listOf(3.0f, 2.0f, 8.0f), listOf(0.0f, 0.0f, 0.0f), listOf(0.0f, 1.0f, 0.0f))

val eye = camera.position()